server/paymentProviders/privacy/index.ts

```diff
@@ -125,7 +153,10 @@ const createExpense = async (
     return expense;
   });
 
-  if (collective.settings?.ignoreExpenseMissingReceiptAlerts !== true) {
+  if (
+    expense && // Expense is undefined if the the transaction is a refund
```

Comment:
You have `isRefund` available as a variable, maybe it would be more explicit to use it?